atuin/src/shell/atuin.nu

```diff
@@ -35,10 +35,14 @@ def _atuin_search_cmd [...flags: string] {
     ] | str join "\n"
 }
 
+let-env config = ($env | default {} config).config
```

Comment:
let-env is going to be removed soon I'm afraid - see https://github.com/ellie/atuin/pull/1080

Reply:
This might be mergeable now. :)